.github/checkgroup.yml

```diff
@@ -190,24 +184,18 @@ subprojects:
     checks:
       - "fabric-cpu (macOS-11, lightning, 3.8, 1.13, oldest)"
       - "fabric-cpu (macOS-11, lightning, 3.10, 1.13)"
-      - "fabric-cpu (macOS-11, lightning, 3.10, 2.0)"
       - "fabric-cpu (macOS-11, lightning, 3.11, 2.1)"
       - "fabric-cpu (ubuntu-20.04, lightning, 3.8, 1.13, oldest)"
       - "fabric-cpu (ubuntu-20.04, lightning, 3.10, 1.13)"
-      - "fabric-cpu (ubuntu-20.04, lightning, 3.10, 2.0)"
       - "fabric-cpu (ubuntu-20.04, lightning, 3.11, 2.1)"
       - "fabric-cpu (windows-2022, lightning, 3.8, 1.13, oldest)"
       - "fabric-cpu (windows-2022, lightning, 3.10, 1.13)"
-      - "fabric-cpu (windows-2022, lightning, 3.10, 2.0)"
       - "fabric-cpu (windows-2022, lightning, 3.11, 2.1)"
       - "fabric-cpu (macOS-11, fabric, 3.8, 1.13)"
       - "fabric-cpu (ubuntu-20.04, fabric, 3.8, 1.13)"
       - "fabric-cpu (windows-2022, fabric, 3.8, 1.13)"
-      - "fabric-cpu (macOS-12, fabric, 3.11, 2.0)"
       - "fabric-cpu (macOS-12, fabric, 3.11, 2.1)"
-      - "fabric-cpu (ubuntu-22.04, fabric, 3.11, 2.0)"
       - "fabric-cpu (ubuntu-22.04, fabric, 3.11, 2.1)"
-      - "fabric-cpu (windows-2022, fabric, 3.11, 2.0)"
```

Comment:
There are 5 removals here but 3 in `ci-tests-fabric.yml`. This doesn't look right. Were there extra lines in the checkgroup config?

Reply:
You are right, I think I should only remove the ones with Python 3.10 + PyTorch 2.0